x/storage/keeper/permission.go

```diff
@@ -236,7 +236,7 @@ func (k Keeper) PutPolicy(ctx sdk.Context, operator sdk.AccAddress, grn types2.G
 			"Only resource owner can put bucket policy, operator (%s), owner(%s)",
 			operator.String(), resOwner.String())
 	}
-
+	k.normalizePrincipal(ctx, policy.Principal)
```

Comment:
Do we need to handle error of this function? There are some errors in this one.

Reply:
No need for this.